Mlem/App/Utility/Extensions/Color+Extensions.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Color+Extensions.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-08-02.
+//
+
+import Foundation
+import SwiftUI
+
+public extension Color {
+    /// Visibly clear color that SwiftUI still renders
+    static var fauxClear: Color { .black.opacity(0.00000000001) }
+}
```

Comment:
A while ago I found that using `Color.clear.contentShape(.rect)` was an alternative solution to using this. Does that work here?

Reply:
It does! Nice, I've always hated this hack